qa/tasks/cephfs/filesystem.py

```diff
@@ -595,7 +595,8 @@ def reset(self):
         self.run_ceph_cmd("fs", "reset", str(self.name), '--yes-i-really-mean-it')
 
     def fail(self):
-        self.run_ceph_cmd("fs", "fail", str(self.name))
+        self.run_ceph_cmd("fs", "fail", str(self.name),
+                          "--yes-i-really-mean-it")
```

Comment:
This will not work when running against an older release (because it's not been backported yet or ever).

I suggest using a `try/except` to avoid the issue.

Reply:
Ah, thanks!